src/components/UI/Navigation/Navigation.js

```diff
@@ -72,6 +72,38 @@ const OuterNav = styled.nav`
   }
 `;
 
+const BreadcrumbWrapper = styled.div`
+  line-height: 20px;
+  background-color: ${({ theme }) => theme.colors.shade3};
+  padding: 7px;
+  border-bottom: ${({ theme }) => theme.colors.shade6} 1px solid;
+  border-top: ${({ theme }) => theme.colors.shade1} 1px solid;
+  font-size: 11px;
+  clear: both;
+
+  @media only screen and (max-width: 1024px) {
+    display: none;
+  }
+
+  a {
+    color: ${({ theme }) => theme.colors.darkBlueLighter6};
+  }
+
+  .breadcrumbs {
+    max-width: 1200px;
```

Comment:
I want to move stuff like this, which is more a global layout property, into the theme, but I'm thinking to do this in a separate PR, in a more considered way.